deployments/helm-chart/crds/appprotect.f5.com_aplogconfs.yaml

```diff
@@ -15,55 +15,51 @@ spec:
   preserveUnknownFields: false
   scope: Namespaced
   versions:
```

Comment:
could you exclude the app protect crds changes?

I added a PR for that https://github.com/nginxinc/kubernetes-ingress/pull/1587